shoes-core/spec/shoes/star_spec.rb

```diff
@@ -202,4 +202,16 @@
       expect(subject.redraw_height).to eq(108)
     end
   end
+
+  describe "center_point" do
+    subject { Shoes::Star.new(app, parent, 100, 100, 5, 50, 30, center: false, strokewidth: 0) }
+
+    it "should return the center point of the star" do
+      expect(subject.center_point.to_a).to eq([150, 150])
```

Comment:
Any particular reason to not just `eq(Shoes::Point.new(150, 150))`?

Also minor point, but you might want to alter dimensions so the center x and y don't match just in case the code got them flopped.

Reply:
No reason at all except my general lack of paying attention.  Thanks for pointing out!  And also, totally good idea about the different coordinates.  Will Fix.